lib/cli/src/commands/create_exe.rs

```diff
@@ -460,6 +442,70 @@ impl CreateExe {
         }
     }
 
+    fn find_filename(
+        local_tarball: &Path,
+        target: &Triple,
+    ) -> Result<(String, PathBuf), anyhow::Error> {
+        let target_file_path = local_tarball
+            .parent()
+            .and_then(|parent| Some(parent.join(local_tarball.file_stem()?)))
+            .unwrap_or_else(|| local_tarball.to_path_buf());
+
+        let target_file_path = target_file_path
+            .parent()
+            .and_then(|parent| Some(parent.join(target_file_path.file_stem()?)))
+            .unwrap_or_else(|| target_file_path.clone());
+
+        let _ = std::fs::create_dir_all(&target_file_path);
```

Comment:
Any reason why the error here is ignored instead of returned?

Reply:
No, missed it. Sometimes I ignore errors if I don't want the command to crash in case something weird happens, but in this case we should crash instantly.